app/auth/views/users.py

```diff
@@ -45,6 +47,43 @@ def post():
     return jsonify(data='post')
 
 
+@auth.route('/login', methods=['POST'])
+def login():
+    json_payload = get_json_from_request()
+    json_has_required_keys(json_payload, ["emailAddress", "password"])
+    email_address = json_payload.get('emailAddress', None)
+    user = User.get_by_email_address(email_address.lower())
+
+    if user is None or (user.supplier and user.supplier.status == 'deleted'):
+        return jsonify(message='User does not exist'), 403
+    elif encryption.authenticate_user(json_payload.get('password', None), user) and user.active:
+        user.logged_in_at = datetime.utcnow()
+        user.failed_login_count = 0
+        db.session.add(user)
+        db.session.commit()
+
+        login_user(user)
+
+        return jsonify(
```

Comment:
In my API branch, I wanted to add the supplier code to the member info so might need to extract this out to a common method.